Article first draft (#135)
hassanobeid1994/tr_b_causal_2020: article/sections/_8_conclusion.tex

\section{Conclusion}
\label{sec:conclusion}

> Travel demand problems aim at forecasting the impact of proposed project and policies. These problems are causal in nature. 

Author: I think it'd be good to always try to stay broad, using transportation as an example but not the only case.
```suggestion
Forecasting the impact of proposed projects and policies is a causal inference problem. 
```

> To our surprise, there has been very little mention of causal inference in the field of travel demand modeling.

Author: Adding a transition from the previous sentence since I suggested removing travel demand modeling from the topic sentence
```suggestion
Since such forecasting is routine throughout public policy in general and transportation in particular, we are surprised that causal inference methods are rarely used in the field of travel demand modeling.
```

\citet{brathwaite_2018_causal} have documented this disconnect and presented an initial framework for addressing it.

> In this chapter, we build up on \citet{brathwaite_2018_causal} and highlight the importance of using causal graphs and 

Author: No need to cite me twice in two lines!
```suggestion
In this chapter, we highlighted the importance of using causal graphs and 
```

causal inference methods in transportation demand modeling efforts.

> We presented a numerical exercise aiming to show the importance of  the data generating process in the estimation 

Author: ```suggestion
We presented a simulation exercise, aiming to show the importance of the data generating process in the estimation 
```

> of effects of interest resulting from external interventions. 

Author: ```suggestion
of causal effects from external interventions. 
```

> We showed, using the same outcome model, that different assumptions about the data generating processes could 

Author: This change is suggested simply because I found myself asking "the same as what?" when reading. Knowing how some people (like myself) skim papers when stressed for time: abstract, intro, conclusion, results, we may want to make the conclusion as self-contained as possible.
```suggestion
We showed, regardless of outcome model, that different assumptions about the data generating processes (i.e., that differing causal graphs) could 
```

lead to bias in estimates of the effect of interest.

> Data generating processes can be illustrated effectively using DAGs as shown by \citet{pearl_1995_causal}.

Author: ```suggestion
Data generating processes can be illustrated effectively using DAGs, as shown by \citet{pearl_1995_causal}.
```

We have shown that DAGs help researchers and practitioners represent their assumptions about the data generating process. 

> We presented a process allowing researchers and practitioners to construct causal graphs based on their expert opinion of the problem at hand. 

Author: ```suggestion
We detailed a process and guiding questions for constructing causal graphs.
Though this process is rooted in subjective opinion and prior knowledge,
```

> We also presented methods for testing the implications encoded in the the practitioners proposed causal graph. 

Author: This suggested edit follows from the last one!
```suggestion
we demonstrate and reference methods for using data to empirically test the implications encoded in one's causal graph. 
```

> These implications might be either observed, latent, or both.

Author: ```suggestion
These implications might involve observed variables, latent variables, or both.
```

> Beyond testing the implications and assumptions encoded in a one's causal graph, we describe why causal discovery is important in discerning relationships between covariates one's data.

Author: ```suggestion
Beyond testing the implications and assumptions encoded in one's causal graph, we described why causal discovery is important for discerning relationships between covariates one's data.
```

> We shortly presented several causal discovery algorithms used by
> other researchers in different fields. 
> We highlighted the use of one of these causal discovery algorithms on a simple example. 

Author: ```suggestion
To provide an introduction to these methods, we briefly presented several representative causal discovery algorithms, and we demonstrated the use of one of these causal discovery algorithms on our chapter's travel mode-choice example. 
```

We showed that causal discovery methods help us test independence within our data, 
develop a clearer picture about the uncertainty within the variables shown in the causal graph, 
and expand the level and amount of tests performed on our data when compared to the tests implicated 

> by the structure of the researcher's proposed causal graph.

Author: I find this sentence to be quite complex! There's the list of three things but your third element is itself a complex, compound thought: "expand the level **and** amount of tests [...] when compared to [...]." This is a lot! I would suggest splitting this sentence, or at least splitting out the third element of the sentence.

Second, what's the difference between "help us test independence within our data" and "expand the level and amount of tests [...]." What additional testing help are your referring to? Also what's the level of the tests?

Lastly, I'd say the three biggest benefits of causal discovery are: better characterized uncertainty about one's causal graph, increased understanding about how one's priors on the graph structure affects one's inferences, and increased scrutiny / alteration of one's own graph in reaction to the discovered graphs. No particular order of importance, or at least I don't have strong feelings about the order.

> We presented examples of latent confounding within the transportation demand modeling field and highlight 
> some examples of current methods aiming at addressing problems resulting from confounding in certain situations.

Author: ```suggestion
We presented examples of latent confounding within the transportation demand modeling field and highlighted
some methods that address latent confounding in specific situations.
```

> We present a recently developed algorithm by \citet{wang_2019_blessings} aiming to address the problems not 
> addressed by previously developed methods dealing with confounding. We use this model on the same example illustrated in
> the selection on observables simulations and highlight the instances where it leads to a better recovery of 
> a confounder in one's causal graph. 

Author: ```suggestion
Specifically, we presented a recently developed algorithm by \citet{wang_2019_blessings}, The Deconfounder.
This algorithm aims to address the problem of latent, multi-cause confounding.
To investigate its usefulness, we applied this technique to the same simulation example mentioned above, and we highlighted the instances where it improved recovery of 
a confounder in our causal graph. 
```

Author: I figured we should use the name of the deconfounder here before referencing it in the next paragraph (assuming not everyone read earlier sections). The edits on line 37 came from wanting a transition from the previous sentence and wanting to avoid explicit reference to the earlier sections.

Author: Also I figured we should avoid even mentioning "the problems not addressed by previous methods" if we're not going to state them explicitly.

> We then show that while the deconfounder algorithm might be a promising step in the right direction, 

Author: ```suggestion
We then show that while the deconfounder algorithm is a promising step in the right direction, 
```

> it still has some notable deficiencies. More specifically, we show that the deconfounder algorithm shows 

Author: ```suggestion
it still has some notable deficiencies. Specifically, we show that the deconfounder algorithm is
```

> to be sensitive to small errors in one's data.

Author: ```suggestion
sensitive to small errors in one's data.
```
